### ISSUE TITLE
performance issue when switch indexdb

### ISSUE THREAD

**Issue by n4mine** [VictoriaMetrics/VictoriaMetrics#801]
**Describe the bug**
slower query duration and high resource usage at 2020-10-01T04:00:00 and last to 2020-10-01T04:10

i think it's relate to switch indexdb

the logs from vmstorage
```
2020-10-01T04:00:00.000Z        info    VictoriaMetrics/lib/mergeset/table.go:169       opening table "/data1/vmdata/indexdb/16378D2E83FAF341"...
2020-10-01T04:00:00.002Z        info    VictoriaMetrics/lib/mergeset/table.go:203       table "/data1/vmdata/indexdb/16378D2E83FAF341" has been opened in 0.00
2 seconds; partsCount: 0; blocksCount: 0, itemsCount: 0; sizeBytes: 0
2020-10-01T04:00:00.003Z        info    VictoriaMetrics/lib/mergeset/table.go:219       waiting for raw items flusher to stop on "/data1/vmdata/indexdb/163031
BEF7343B5F"...
2020-10-01T04:00:00.003Z        info    VictoriaMetrics/lib/mergeset/table.go:222       raw items flusher stopped in 0.000 seconds on "/data1/vmdata/indexdb/1
63031BEF7343B5F"
2020-10-01T04:00:00.003Z        info    VictoriaMetrics/lib/mergeset/table.go:224       waiting for converters to stop on "/data1/vmdata/indexdb/163031BEF7343
B5F"...
2020-10-01T04:00:00.003Z        info    VictoriaMetrics/lib/mergeset/table.go:227       converters stopped in 0.000 seconds on "/data1/vmdata/indexdb/163031BE
F7343B5F"
2020-10-01T04:00:00.003Z        info    VictoriaMetrics/lib/mergeset/table.go:229       waiting for part mergers to stop on "/data1/vmdata/indexdb/163031BEF73
43B5F"...
2020-10-01T04:00:00.003Z        info    VictoriaMetrics/lib/mergeset/table.go:232       part mergers stopped in 0.000 seconds on "/data1/vmdata/indexdb/163031
BEF7343B5F"
2020-10-01T04:00:00.004Z        info    VictoriaMetrics/lib/mergeset/table.go:234       flushing inmemory parts to files on "/data1/vmdata/indexdb/163031BEF73
43B5F"...
2020-10-01T04:00:00.004Z        info    VictoriaMetrics/lib/mergeset/table.go:258       0 inmemory parts have been flushed to files in 0.000 seconds on "/data
1/vmdata/indexdb/163031BEF7343B5F"
2020-10-01T04:00:00.005Z        info    VictoriaMetrics/lib/storage/index_db.go:380     dropping indexDB "/data1/vmdata/indexdb/163031BEF7343B5F"
2020-10-01T04:00:00.108Z        info    VictoriaMetrics/lib/storage/index_db.go:382     indexDB "/data1/vmdata/indexdb/163031BEF7343B5F" has been dropped
2020-10-01T04:00:00.117Z        error   VictoriaMetrics/lib/storage/index_db.go:2842    missing metricName by metricID 1598832563978513058; this could be the case after unclean shutdown; deleting the metricID, so it could be re-created next time

and more 2193 lines like `missing metricName by metricID ...; this could be the case after unclean shutdown; deleting the metricID, so it could be re-created next time`
```

**To Reproduce**
at 2020-10-01T04:00:00(UTC)

**Expected behavior**
not more slow query and high resource ussage

**Screenshots**
![111](https://user-images.githubusercontent.com/3345153/94769961-e50a8c80-03e5-11eb-9800-265c0fdbd8a8.jpg)

**Version**
The line returned when passing `--version` command line flag to binary. For example:
```
$ ./vmstorage-prod -version
vmstorage-20200923-230611-tags-v1.41.1-cluster-0-g90d25494
```

**Comment by f41gh7** [comment]
It looks like an issue with noneffective cache for requests to previous inverted index.  It was fixed at master with commit https://github.com/VictoriaMetrics/VictoriaMetrics/commit/10f2eedee08580670f09744d59d7cc904126f1bf.

**Comment by valyala** [comment]
@n4mine , the issue should be fixed after the commit fd7dd5064a5a327f4ad6cddff0dbb4c9b9472783 of [cluster branch](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/cluster).

**Comment by n4mine** [comment]
@valyala @f41gh7 
thanks, i'll be upgrade later.
